agroal-api/src/main/java/io/agroal/api/configuration/AgroalConnectionFactoryConfiguration.java

```diff
@@ -21,7 +21,7 @@
 
     Class<?> connectionProviderClass();
 
-    TransactionIsolation jdbcTransactionIsolation();
+    IsolationLevel jdbcTransactionIsolation();
```

Comment:
This breaks binary compatibility, if that matters.

Reply:
I don't think it matters. This interface is not intended to be called/implemented outside agroal.